benchopt/utils/dynamic_modules.py

```diff
@@ -123,3 +138,48 @@ def _reconstruct_class(module_filename, class_name, benchmark_dir,
         )
 
     return _load_class_from_module(module_filename, class_name, benchmark_dir)
+
+
+def _get_cls_attributes(module_file, cls_name):
+    module = ast.parse(module_file.read_text())
+
+    cls_list = [node for node in module.body if isinstance(node, ast.ClassDef)
+                                           and node.name == cls_name]
+    if not cls_list:
+        raise ValueError(f"Could not find {cls_name} in module {module_file}.")
+    cls = cls_list[0]
+
+    for node in cls.body:
```

Comment:
Sometime, these attributes are not defined. We should raise comprehensive errors.
```suggestion
    name, install_cmd, reqs = None, "conda", []
    for node in cls.body:
```